src/js/angular/clustermanagement/templates/cluster-configuration/multi-region.html

```diff
@@ -1 +1,146 @@
-multi-region
+<div onto-loader-new ng-if="loader" class="create-cluster-loader" message="getLoaderMessage" size="100  "></div>
+
+<div class="multi-region">
+    <div ng-if="topology.state === TopologyState.PRIMARY_NODE">
+        <h4>{{'cluster_management.cluster_configuration_multi_region.primary_cluster' | translate}}</h4>
+        <div class="mb-2" ng-if="isAdmin">
+            <button type="button" class="btn btn-primary create-tag-btn"
+                    ng-click="add()"
+                    gdb-tooltip="{{'cluster_management.cluster_configuration_multi_region.create_tag_tooltip' | translate}}"
+                    tooltip-placement="bottom">
+                <i class="fa-regular fa-plus"></i> {{'cluster_management.cluster_configuration_multi_region.create_tag' | translate}}
+            </button>
+            <button type="button" class="btn btn-secondary enable-secondary-mode-btn mr-1"
+                    ng-click="enableSecondaryMode()"
+                    gdb-tooltip="{{'cluster_management.cluster_configuration_multi_region.enable_secondary_mode_tooltip' | translate}}"
+                    tooltip-placement="bottom">
+                <i class="fa-regular fa-circle-arrow-down-right"></i>
+                {{'cluster_management.cluster_configuration_multi_region.enable_secondary_mode' | translate}}
+            </button>
+        </div>
+        <div class="table-responsive" ng-if="topology.primaryTags.length || addingTag">
+            <form name="tagForm">
+                <table class="tags-table table-striped table-hover table" aria-describedby="tags table">
+                    <thead>
+                    <tr class="labels-row">
+                        <th scope="col" class="index-column">
+                            {{'cluster_management.cluster_configuration_multi_region.column.index' | translate}}
+                        </th>
+                        <th scope="col" class="tag-column">
+                            {{'cluster_management.cluster_configuration_multi_region.column.tag' | translate}}
+                        </th>
+                        <th scope="col" class="tag-index-column">
+                            {{'cluster_management.cluster_configuration_multi_region.column.tag_index' | translate}}
+                        </th>
+                        <th scope="col" class="actions-column"></th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr class="add-tag-row" ng-if="addingTag">
+                        <td class="index-cell"></td>
+                        <td class="tag-name-cell">
+                            <input type="text"
+                                   id="asciiInput"
+                                   name="asciiInput"
+                                   ng-model="asciiInput"
```

Comment:
I'm pretty sure that you don't need this `ng-model` here, because angular will automatically bind the input name to the `tagForm` in the $scope.

Reply:
the validators need it